### PR TITLE
Add a script to change the version of odh in the Dockerfile

### PR DESCRIPTION
**Pull request by tmckayus**
This script has been added to support automation.

### CODE REVIEW

**Comment by tmckayus** [comment]
@vpavlin @crobby @anishasthana ptal

maybe a little overkill but the framework is there if we need other options :)

File: odh-operator-image/change-version.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+if [ "$#" -lt 1 ]; then
+    echo error: no odh version specified
+    usage
+    exit 1
+fi
+
+sed -i "s/ARG version=.*/ARG version=$1/" Dockerfile
+git add Dockerfile
+if [ "$commit" == "true" ]; then
+    git commit -m "Change the odh version to $1"
```
Comment (vpavlin) [inline odh-operator-image/change-version.sh]: Can you `s/odh/ODH/` please?

File: odh-operator-image/change-version.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#/bin/bash
+
+function usage() {
+    echo "Modifies the odh version in Dockerfile and calls git add"
```
Comment (vpavlin) [inline odh-operator-image/change-version.sh]: `s/odh/ODH/`

File: odh-operator-image/change-version.sh [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    echo
+    echo "usage: change-version.sh [options] VERSION"
+    echo
+    echo "required arguments:"
+    echo
+    echo "  VERSION    an odh version tag which identifies an opendatahub-operator base image"
```
Comment (vpavlin) [inline odh-operator-image/change-version.sh]: `s/odh/ODH/`

**Pull request merged**